shell/common/shell_unittests.cc

```diff
@@ -479,5 +483,40 @@ TEST_F(ShellTest, ReportTimingsIsCalledLaterInReleaseMode) {
 #endif
 }
 
+TEST_F(ShellTest, ReportTimingsIsCalledImmediatelyAfterTheFirstFrame) {
+  auto settings = CreateSettingsForFixture();
+  std::unique_ptr<Shell> shell = CreateShell(std::move(settings));
+
+  // Create the surface needed by rasterizer
+  PlatformViewNotifyCreated(shell.get());
+
+  auto configuration = RunConfiguration::InferFromSettings(settings);
```

Comment:
How is this working? You just moved the settings into the CreateShell call? Either make it `const` or don't move it.

Reply:
Oops, it seems that I forgot to remove `std::move` when I removed `&&` in the last PR. By the fact that all tests have passed in these days, it seems to me that `std::move` may just be a no-op here?

Anyway, `std::move` removed.